tests/modules/services/pantalaimon/basic-configuration.nix

```diff
@@ -0,0 +1,27 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.pantalaimon = {
+      enable = true;
+      package = pkgs.writeScriptBin "dummy-pantalaimon" "" // { outPath = "@pantalaimon@"; };
+      settings = {
+        Default = {
+          LogLevel = "Debug";
+          SSL = true;
+        };
+        local-matrix = {
+          Homeserver = "https://matrix.org";
+          ListenAddress = "127.0.0.1";
+          ListenPort = 8008;
+        };
+      };
+    };
+
+    nmt.script = ''
+      serviceFile=home-files/.config/systemd/user/pantalaimon.service
+      assertFileExists $serviceFile
+      assertFileRegex $serviceFile 'ExecStart=@pantalaimon@/bin/pantalaimon -c /nix/store/.*-pantalaimon.conf'
```

Comment:
Technically, the generated path should always be the same but it doesn't matter.